docs/reference/api/storage/ProfilingBlockDevice.md

```diff
@@ -1,2 +1,15 @@
 ## ProfilingBlockDevice
 
+The ProfilingBlockDevice class provides a BlockDevice implementation to wrap around an existing block device object to log reads, writes and erases.
```

Comment:
Content query: Is this too much like calling this a wrapper?

Reply:
To be more precise the ProfilingBlockDevice class is a [decorator](https://en.wikipedia.org/wiki/Decorator_pattern) which adds profiling capabilities to a BlockDevice instance. 

@sg- What is our position on design pattern terminology ? Should we use it or avoid it ?

Reply:
We should use it.